### PR TITLE
Add Hungarian translation

### PR DESCRIPTION
**Pull request by smileyhead**
Hello there.

### CODE REVIEW

**Comment by smileyhead** [comment]
Like this?

File: locales/hu.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+  "general": {
+    "daisy-mae": "Daisy Mae"
+  },
+  "welcome": {
+    "salutation": "Üdvözöllek a Nookfonod <b>Retekpróféta</b> alkalmazásában!",
```
Comment (theRTC204) [inline locales/hu.json:6]: Please leave uses of the name "Turnip Prophet" without translation. We are treating it as a proper name which would not be translated.

File: locales/hu.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    "minimum": "Garantált minimum",
+    "maximum": "Lehetséges maximum",
+    "chart": {
+      "input": "Megadott ár",
+      "minimum": "Garantált minimum",
+      "maximum": "Lehetséges maximum"
+    }
+  },
+  "textbox": {
+    "description": "Miután megadtál néhány árat, a Retekpróféta kiszámolja és megjeleníti az összes lehetséges mintát ami a héten megjelenhet.",
```
Comment (mikebryant) [inline locales/hu.json:72]: App name needs to be kept here too :)

**Comment by smileyhead** [comment]
Oh, sorry about that. Fixed.

**Pull request merged**